wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUFileDetailInfo.java

```diff
@@ -86,12 +87,14 @@ public void setFirstline(String firstline)
     public String toString()
     {
         StringBuilder sb = new StringBuilder();
-        if (this.getActualSize() != null)
-            sb.append("ActualSize:").append(this.getActualSize()).append("\n");
+        //removed in HPCC 5.0.0
```

Comment:
Have these functions that apparently were deprecated since 5.0.0 caused any issues? Will their absence cause any problems with how people expect toString() to return output?

Reply:
We'll notify known stakeholders, and on the README. DSP and Hipie confirmed they are completely off of 5.x